pifm.cpp

```diff
@@ -202,7 +203,7 @@ class Outputter : public SampleSink {
     };
     void consume(float* data, int num) {
         for (int i=0; i<num; i++) {
-            float value = data[i]*8;  // modulation index (AKA volume!)
+            float value = data[i]*4*volume;  // modulation index (AKA volume!)
```

Comment:
should that be `4*volume` or just `volume` ?

if the former, should the default be 2?
if the latter, should the default be 8?